### PR TITLE
Effects, includes

### PR DESCRIPTION
**Pull request by ethteck**
Some effect work/cleanup + deduplication

### CODE REVIEW

**Comment by BowserSlug** [comment]
🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀 This PR matches 84 functions (+2.47%) on `us`.

✅ This PR fixes 74 warnings!

File: include/effects.h [hunk 1]
```diff
@@ -168,6 +222,108 @@ typedef struct Effect12 {
...
+    /* 0x38 */ s32 unk_38;
+    /* 0x3C */ s32 unk_3C;
+    /* 0x40 */ s32 unk_40;
+    /* 0x44 */ s32 unk_44;
+} Effect13; // size = 0x48
+
+typedef struct Effect14 {
+    /* 0x00 */ f32 xPos; //xPos
+    /* 0x04 */ f32 yPos; //yPos
+    /* 0x08 */ f32 zPos; //zPos
```
Comment (nanaian) [inline include/effects.h:249]: Redundant comments

File: src/world/common/UnkPosFunc2.inc.c [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+        script->varTable[2] = 1;
+    }
+
+    for (i = 0; i < 0x40; i++) {
+        Npc* npc = get_npc_safe(i);
+
+        if (npc != NULL) {
+            xDiff = npc->pos.x - model->center.x;
+            zDiff = npc->pos.z - model->center.z;
+            yVal = ((npc->pos.y + npc->collisionHeight) - 1.5f) - model->center.y;
```
Comment (nanaian) [inline src/world/common/UnkPosFunc2.inc.c:40]: Should be able to remove all the parens
```suggestion
            yVal = npc->pos.y + npc->collisionHeight - 1.5f - model->center.y;
```

File: src/world/common/UnkPosFunc2.inc.c [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+    yVal = playerStatus->position.y + playerStatus->colliderHeight - 1.5f - model->center.y;
+    sqrtTemp = sqrtf(SQ(xDiff) + SQ(zDiff));
+
+    if (yVal > 0.0f && yVal < playerStatus->colliderHeight && sqrtTemp < playerStatus->colliderDiameter * 0.5f) {
+        script->varTable[2] = 1;
+    }
+
+    xDiff = partner->pos.x - model->center.x;
+    zDiff = partner->pos.z - model->center.z;
+    yVal = ((partner->pos.y + (f32) partner->collisionHeight) - 1.5f) - model->center.y;
```
Comment (nanaian) [inline src/world/common/UnkPosFunc2.inc.c:27]: "

**Review by nanaian — CHANGES_REQUESTED** [review]

**Pull request merged**